core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -110,7 +110,14 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging {
 
     def abortQueuedWorkflows(submissionId: UUID) = {
       datasource.inTransaction { dataAccess =>
-        dataAccess.workflowQuery.batchUpdateWorkflowsOfStatus(submissionId, WorkflowStatuses.Queued, WorkflowStatuses.Aborted)
+        // Update workflows in batches of 10 to limit database deadlocks between
+        // this actor and WorkflowSubmissionActor. See GAWB-2243.
+        dataAccess.workflowQuery.listWorkflowRecsForSubmissionAndStatuses(submissionId, WorkflowStatuses.Queued).flatMap { workflows =>
+          val grouped = workflows.grouped(10).map { wfs =>
```

Comment:
I don't think we have a hard policy here. Everything in rawls.conf can override anything in ref.conf but that is a relatively recent fact which means almost all of our config is in rawls.conf. But now it does not really matter anymore. I would be happy if we had a move to put all relatively static things in ref.conf and remove them from rawls.conf.